### DISCUSSION TITLE
Anyone else had issues with latest version?

### DISCUSSION THREAD

**Discussion by djenson2000** [wills106/homeassistant-solax-modbus#796] [General]
I have a Solax X1-Hybrid-G4 - which was working fine with the latest version of the integration. I then installed something from HACS that required a restart, and upon restart, there was no data, and in logs a "couldn't find serial number" or something similar. I tried reconnecting from the integration, and when that failed, i pulled the dongle out to reset it and tried connecting again, still no joy. i ended up restoring a backup with version 2024.02.6 and everything was fine. Thinking it may have been the additional install from HACS that screwed everything up, i reinstalled it, but everything's working fine still.

**Comment by mikesask** [comment]
Yes, no luck for me, restoring as we speak, shame they don't have an update reverse or something make life easier.
